.github/workflows/deploy-to-eks.yaml

```
@@ -121,7 +127,14 @@ jobs:
           git clone https://github.com/canonical/charmed-kubeflow-uats.git ~/charmed-kubeflow-uats
           cd ~/charmed-kubeflow-uats
           git checkout ${{ env.UATS_BRANCH }}
-          tox -e kubeflow-remote
+          
+          EXTRA_ARGS=""
+          if [[ -z "${{env.BUNDLE_URL}}" ]];
+          then 
+            EXTRA_ARGS="-- --bundle-url ${{env.BUNDLE_URL}}"
+          fi
```

```suggestion
EXTRA_ARGS=${BUNDLE_URL:+-- --bundle-url $BUNDLE_URL}
```
just a suggestion, probably not as readable. Feel free to ignore.